packages/react_rearch/lib/src/widgets/consumer.dart

```diff
@@ -0,0 +1,158 @@
+part of '../widgets.dart';
+
+abstract class RearchComponent extends Component2 {
+  bool _building = false;
+
+  String get debugName;
+
+  bool get debug => false;
+
+  void _debug(String msg) {
+    if (!debug) return;
+    // ignore: avoid_print
+    print('<$debugName> -- $msg');
+  }
```

Comment:
Could you remove all of this debug-related code? I tend to not prefer devs to have to override debug related info unless they want to